src/Vandelay.cpp

```diff
@@ -1,26 +1,45 @@
-#include "VandelayWindow.h"
+
 #include "Vandelay.h"
 
+const char * kSignature = "application/x-vnd.Blum-Vandelay";
 
-int main(int, char**)
-{	
-	VandelayApp VanApp;
 
-	VanApp.Run();
-
-	return(0);
+VandelayApp::VandelayApp() : BApplication(kSignature) {
+	
+	
```

Comment:
I should have updated it to conform to the coding style straight away instead of keeping it in TODO. :) I'll do that.